tests/unit_tests/commandline/cmd_instructions/invalid_json/cmd_instructions.json

```diff
@@ -1,6 +1,6 @@
 {
-    "name": "I_am_invalid",
-    "because_my_name_is_not_instructions": {
+    "name": "geoips_cmd_instructions",
+    "instructions": {
```

Comment:
I have noticed this file gets modified sometimes - was this supposed to be checked in?

Reply:
It is supposed to be checked in. This is used for the unit tests. I'm happy just removing the test cases under ``tests/unit_tests/commandline/cmd_instructions`` as they make testing more complex and cause problems with git. Up to you on this one though.